src/use-swr.ts

```diff
@@ -232,6 +246,7 @@ function useSWR<Data = any, Error = any>(
 
   const initialData = cache.get(key) || config.initialData
   const initialError = cache.get(keyErr)
+  const initialIsValidating = cache.get(keyValidating)
```

Comment:
small question, not blocking, will the `initialIsValidating` become undefined at the first time? do we want to forcedly convert it to a boolean value here like `Boolean(cache.get(keyValidating))` or `!!cache.get(keyValidating)`?

Reply:
We can convert it to a boolean. Probably a good practice.

Reply:
Done